src/article/editor/InsertNodeFromWorkflowCommand.js

```diff
@@ -0,0 +1,12 @@
+import InsertNodeCommand from './InsertNodeCommand'
+
+export default class InsertNodeFromWorkflowCommand extends InsertNodeCommand {
+  execute (params, context) {
+    const workflow = this.config.workflow
+    if (workflow) {
+      context.editor.send('startWorkflow', workflow)
+    } else {
```

Comment:
Explain what the else-branch is used for. ATM it seems not be used. If you do not have good reasons, remove it.